cocos/tween/actions/action-manager.ts

```diff
@@ -111,12 +108,6 @@ export class ActionManager {
             return;
         }
 
-        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-        // @ts-expect-error
-        if (target.uuid == null) {
-            (target as any).uuid = `_TWEEN_UUID_${ID_COUNTER++}`;
-        }
```

Comment:
Will it break compatibility?

Reply:
NO，I think it was added just for debugging purpose. Nobody will use it.
And what's more, it modify the Hidden Class  of target object which should be forbiden.

Reply:
Ok.